tox.ini

```diff
@@ -14,7 +14,8 @@ skipsdist = true
 changedir = tests
 
 commands =
-    pylint {toxinidir}/tuf
+    pylint {toxinidir}/tuf --ignore={toxinidir}/tuf/api
+    pylint {toxinidir}/tuf/api --rcfile={toxinidir}/tuf/api/pylintrc
```

Comment:
What about https://github.com/theupdateframework/tuf/pull/1157#issuecomment-702025224? Or do you purposefully want to be explicit here?

Reply:
Also [looks](https://travis-ci.org/github/theupdateframework/tuf/builds/736003163) like we can only run pylint on supported versions for `tuf.api`.

Reply:
I wonder if the linter really needs to run for every supported version. In [in-toto we have a separate lint tox env](https://github.com/in-toto/in-toto/blob/c775e3b260a50ec5fb673a9b725e2d2eacedfb6b/tox.ini#L22-L25) that uses the latest supported Python version.

Reply:
> What about [#1157 (comment)](https://github.com/theupdateframework/tuf/pull/1157#issuecomment-702025224)? Or do you purposefully want to be explicit here?

It didn't seem to work when I tried it. May be because cwd isn't a Python module when we invoke pylint? Being explicit seemed easier than relying on pylint magic I couldn't fully decipher from the documentation.

> I wonder if the linter really needs to run for every supported version. In in-toto we have a separate lint tox env that uses the latest supported Python version.

This feels like a sensible option, I'll add a separate lint env.